### ISSUE TITLE
"Continue: Rebuild codebase index" command not available in JetBrains plugin?

### ISSUE THREAD

**Issue by dandv** [continuedev/continue#3557]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- Continue version: 0.0.83
- IDE version: WebStorm 2024.2.4 Build #WS-242.23726.96, built on October 22, 2024
```

### Description

https://docs.continue.dev/troubleshooting#indexing-issues says

> This can be accomplished using the following command: <kbd>Continue: Rebuild codebase index</kbd>.

However, I don't see that command.

### To reproduce

1. Press Ctrl+Shfit+A
2. Type `continue`

![image](https://github.com/user-attachments/assets/fafb5b4e-546d-4d66-8710-867ef8d87d45)

**Comment by sestinj** [comment]
yup, good catch this is only in VS Code. You can use the options in the "More" page of the sidebar to do this as well until we add the command. Going to leave this as good-first-issue
